src/views/tutorial-view/components/next-previous/components/directional-link-box/types.ts

```diff
@@ -0,0 +1,8 @@
+export type DirectionOption = 'next' | 'previous' | 'final'
+
+export interface DirectionalLinkBoxProps {
+  href: string
+  label: string
+  title: string
```

Comment:
Thoughts on referencing `CardLinkProps['href']` for `href` and `CardLinkProps['ariaLabel']` for `title` (potentially renaming to `ariaLabel`)?

Reply:
I'm on board with it! Have made the change since I don't have a strong opinion 👍

My weak opinion: Personally I don't see a huge difference either way - I _think_ TypeScript would warn us if the `string` type we declare here was incompatible with `CardLinkProps`. I slightly prefer the transparency of saying `string` rather than the relative opacity of `CardLinkProps['href']`. Super minor and weak opinion though, seeing that's it's `string` is just a hover away so 🤷‍♂️